### ISSUE TITLE
[NTR]: spermatozoon

### ISSUE THREAD

**Issue by mikewhitelab** [Tree-of-Sex/tos-ontology#51]
### In which category is your new term request?

- [ ] new_definition
- [X] existing_definition

### Preferred term label

spermatozoon

### Synonyms

spermatozoa
sperm

### Textual definition

A mature male germ cell that develops from a spermatid.
http://purl.obolibrary.org/obo/XAO_0003147

### Suggested parent term

Spermatogenesis
gamete

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by laiamaringual** [comment]
> ### Preferred term label
> 
> spermatozoon
> ### Synonyms
> 
> spermatozoa sperm
> ### Textual definition
> 
> A mature male germ cell that develops from a spermatid. http://purl.obolibrary.org/obo/XAO_0003147
> ### Suggested parent term
> 
> Spermatogenesis gamete
> ### Attribution
> 
> _No response_
> ### Additional context
> 
> _No response_
> ### Code of Conduct


I agree with the definition

**Comment by lauuragon** [comment]
I agree with this definition: A mature male germ cell that develops from a spermatid.
http://purl.obolibrary.org/obo/XAO_0003147
